### PR TITLE
DX-2557: Enable release drafter

### PR DESCRIPTION
**Pull request by danepowell**
Motivation
----------
Creating release notes consistently is a pain

Proposed changes
---------
Use https://github.com/release-drafter/release-drafter to automatically generate release notes based on PR labels:
- Breaking / major changes (major version increment)
- Enhancements (minor version increment)
- Bug fixes (patch version increment)

New releases will be kept in a draft state and constantly updated with new PRs until a human publishes them.

Alternatives considered
---------
We could split control of the issue type and severity. So have enhancement / bug labels in addition to major / minor / patch labels for direct control over versioning.

Testing steps
---------
I don't think this can be tested until it's merged 🤷

### CODE REVIEW

**Review by anavarre — APPROVED** [review]
Approved to unblock you. As you pointed out, if this cannot be tested until merged, let's do it and see.

**Comment by danepowell** [comment]
Sweet, as soon as I merged this the script ran and it looks pretty good! Let me know if you have feedback @anavarre or @grasmash : https://github.com/acquia/cli/releases/tag/untagged-12d0d1e8f1dedcff2823

**Comment by anavarre** [comment]
No feedback except it's perfect :+1:

**Comment by grasmash** [comment]
Looks great. Rainbow is a bit much.

**Comment by anavarre** [comment]
Would you prefer confetti? :wink:

**Comment by danepowell** [comment]
Hah. Rainbow was in the template, I didn't even notice.

**Pull request merged**
